system/jhtree/ctfile.hpp

```diff
@@ -278,7 +309,7 @@ class CJHVarTreeNode : public CJHTreeNode
     virtual int compareValueAt(const char *src, unsigned int index) const;
 };
 
-class CJHRowCompressedNode : public CJHTreeNode
+class CJHRowCompressedNode : public CJHExpandedTreeNode
```

Comment:
I thought rowCompressed could be read without expanding

Reply:
Yes it can.  For the moment I would leave it like this, but it probably would make sense to refactor it later.